jdk/make/CompileLaunchers.gmk

```diff
@@ -104,6 +104,17 @@ define SetupLauncher
   endif
   
   $1_LDFLAGS_SUFFIX :=
+
+  # Set text/data load address for x64_aix to 0x100,0000,0000 (1TB) for launchers,
```

Comment:
I don't think `x64` is appropriate for AIX. x64 usually means Intel, while AIX uses Power PC.